src/shared/components/header/header.css

```diff
@@ -0,0 +1,162 @@
+.nav-container{
+  display: flex;
+  min-height: 12vh;
+  padding:0.8rem 1.2rem 0.8rem 0.8rem;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 2px 2px 5px rgb(0 0 0 / 20%);
+  background-color:var(--light-color);
+  position: sticky;
+  top:0;
+  z-index:10;
+  width:100%;
+}
+.logo {
+  margin: 0 1rem;
+}
+.left-navbar {
+  display: flex;
+}
+.navbar-link-list-item {
+  margin: 0 1rem;
+}
+.right-navbar {
+  display: flex;
+  align-items:center;
+}
+.navbar-links {
+  display: flex;
+  align-items: center;
+}
+.badge-icon{
+  font-size: 1.3rem ;
+}
+.search-bar-container {
+  display: flex;
+  align-items: center;
+  margin:0 0.5rem;
+  background-color: rgb(221, 221, 221);
+  border-radius: 0.3rem;
+}
+.search-bar-container button {
+  height: 2rem;
+  padding: 0 0.5rem;
+  margin: 0;
+  border: none;
+  background-color: transparent;
+}
+.search-bar {
+  height: 2rem;
+  background-color: transparent;
+  border: none;
+  outline: 0px;
+}
+.nav-icons {
+  display: flex;
+}
+.nav-icon-link {
+  align-items: center;
+  border:none;
+  background-color:transparent;
+}
+.dropbtn {
+  color: var(--dark-color);
+  font-size: 1rem;
+  border: none;
+}
+
+.dropdown {
+position: relative;
+display: inline-block;
+}
+/* Dropdown Content*/
+.dropdown-content {
+display: none;
+position: absolute;
+top: 3rem;
+left: 0rem;
+background-color: var(--light-color);
+min-width: 10rem;
+box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+z-index: 1;
+}
+/* Links inside the dropdown */
+.dropdown-content a {
+color: var(--dark-color);
+padding: 12px 16px;
+text-decoration: none;
+display: block;
+}
+.dropdown-content a:hover {
+  background-color: var(--dropdown-bg);
+}
+
+/* Show the dropdown menu on hover */
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+.dropdown:hover .dropbtn {
+color: var(--dark-pink);
+}
+
+.wishlist-button{
+    margin: 0rem  2.5rem;
+}
+.icon-size {
+  font-size: 1.3rem;
+}
+.hamburger-icon{
+    display: none;
+}
+.text-xs-sm{
+  font-size: 1.3rem;
+}
+
+@media only screen and (max-width:800px){
+  *{
+      font-size: 80%;
+  }
+  .hamburger-icon{
+      font-size: 1.5rem;
+  }
+  .navbar-link-list-item{
+      font-size: 1.3rem;
+  }
+  .nav-list-link{
+      font-size: 1rem;
+  }
+
+}
+@media only screen and (max-width:768px) {
+  .navbar-container{
+      padding:0.8rem
+  }
+  .logo{
+      display: none;
+  }
+  .hamburger-icon{
+      display: block;
+  }
+  
+  
+ 
+}
+@media only screen and (max-width:600px) {
+  .navbar-link-list-item{
+      display: none;
+      font-size: 1.5rem;
+  }
+  .badge-icon{
+      font-size: 1rem !important;
```

Comment:
why use important here. If it is at last then this will be applied

Reply:
as I have media queries which have particular font-size but I want to override hence I have to mark it important !!

Reply:
It is at last but I want this specific font-size hence I have to mark it as important !!